_security-plugin/configuration/multi-auth.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: default
+title: Multiple option authentication
+parent: Configuration
+nav_order: 3
+---
+
+# Multiple option authentication for Dashboards sign-in
```

Comment:
just an optional suggestion. . .could it be more direct?

"Configure Dashboards sign-in options"